packages/debug/src/browser/view/debug-configuration-select.tsx

```diff
@@ -0,0 +1,209 @@
+/********************************************************************************
+ * Copyright (C) 2021 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import URI from '@theia/core/lib/common/uri';
+import * as React from '@theia/core/shared/react';
+import { DebugConfigurationManager } from '../debug-configuration-manager';
+import { DebugSessionOptions, InternalDebugSessionOptions } from '../debug-session-options';
+import { QuickInputService } from '@theia/core/lib/browser';
+import { nls } from '@theia/core/lib/common/nls';
+
+interface IDynamicPickItem { label: string, configurationType: string, request: string, providerType: string }
```

Comment:
The `I` prefix [is discouraged](https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#interfaces-no-i-prefix)
```suggestion
interface DynamicPickItem { label: string, configurationType: string, request: string, providerType: string }
```